src/util/useCheckCanDelete.ts

```diff
@@ -0,0 +1,17 @@
+import useOktaTokens from "../hooks/useOktaTokens";
+
+const useCheckUserCanDelete = (
+  createdBy: string,
+  draft: boolean = true
+): boolean => {
+  const { getUserName } = useOktaTokens();
```

Comment:
`const { loggedInUser } = useOktaTokens();`

Reply:
Ashok,  these are the only function that are present in the useOktaToken I cannot change the name to loggedInUser

useOktaTokens(storageKey?: string): {
    getAccessToken: () => any;
    getAccessTokenObj: () => any;
    getUserName: () => any;
    getIdToken: () => any;
    getIdTokenObj: () => any;
}

Reply:
my bad. yeah, you are right. I think I meant to change it next line. `const loggedInUser = getUserName();`